pkg/subi/errors.go

```diff
@@ -6,13 +6,17 @@ import (
 	subv3 "github.com/threefoldtech/substrate-client-test"
 )
 
+// ErrNotFound is an error for substrate not found
 var ErrNotFound = subv2.ErrNotFound
+
+// ErrAccountNotFound is an error for substrate account not found
 var ErrAccountNotFound = subv2.ErrAccountNotFound
 
-func terr(err error) error {
+func isNotFoundErrors(err error) error {
```

Comment:
problem with the name is it's confusing, because functions starting with `is` imply they return boolean

Let's create `isNotFoundError(err error) bool` that returns true in case of `ErrAccountNotFound` or `ErrNotFound`

and we can create `normalizeNotFoundErrors` to return our custom errors or return the error as is